cmd/healthcheck-handler.go

```diff
@@ -106,6 +106,17 @@ func LivenessCheckHandler(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set(xhttp.MinIOServerStatus, unavailable)
 	}
 
+	if int(globalHTTPStats.loadRequestsInQueue()) > globalAPIConfig.getRequestsPoolCapacity() {
```

Comment:
Checking if len(t.requestsPool) == cap(t.requestsPool), if non zero, can work as well. You may like it better but still LGTM.

Reply:
IMO for readability this is what made sense to me @vadmeste